doc/man3/DH_generate_parameters_ex.pod

```diff
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-DH_generate_parameters_ex, DH_generate_parameters,
+DH_generate_parameters, DH_generate_parameters_ex,
```

Comment:
I'd leave it the way it was, i.e., the not quite so deprecated version first.

Reply:
good point.  fixed.